src/cgame/cg_main.cpp

```diff
@@ -53,6 +53,17 @@ Cvar::Range<Cvar::Cvar<int>> cg_drawClock("cg_drawClock", "draw clock (1 = 12-ho
 Cvar::Cvar<bool> cg_drawFPS("cg_drawFPS", "show client's frames per second", Cvar::NONE, true);
 Cvar::Range<Cvar::Cvar<int>> cg_drawCrosshair("cg_drawCrosshair", "draw crosshair (1 = ranged weapons, 2 = always)", Cvar::NONE, 2, 0, 2);
 Cvar::Cvar<bool> cg_drawCrosshairHit("cg_drawCrosshairHit", "show damage indicator", Cvar::NONE, true);
+Cvar::Range<Cvar::Cvar<int>> cg_crosshairStyle( "cg_crosshairStyle", "crosshair colour style (0 = image colour, 1 = custom colour)", Cvar::NONE, 0, 0, 1 );
```

Comment:
Can this be an Enum for readability?

Reply:
I don't mean make the cvar an enum, but do comparisons against enums

Reply:
Done